types/stripe-js/index.d.ts

```diff
@@ -300,6 +328,23 @@ declare module '@stripe/stripe-js' {
     /// https://stripe.com/docs/js/setup_intents
     /////////////////////////////
 
+    /**
+     * Use `stripe.confirmAcssDebitSetup` in the [Save bank details](/docs/payments/acss-debit/set-up-payment) flow to set up a [Canadian pre-authorized debit](/docs/payments/acss-debit) payment method for future payments.
```

Comment:
I think both these links can be replaced with the single link https://stripe.com/docs/payments/acss-debit/set-up-payment